fastai/vision/augment.py

```diff
@@ -575,27 +663,53 @@ def _def_draw_b(x): return random.randint(0,7) + x.new_zeros((x.size(0),)).long(
 
 # Internal Cell
 @patch
-def dihedral_batch(x: (TensorImage,TensorMask,TensorPoint,TensorBBox), p=0.5, draw=None, size=None,
-                   mode=None, pad_mode=None, batch=False, align_corners=True):
+def dihedral_batch(x: (TensorImage,TensorMask,TensorPoint,TensorBBox),
+    p=0.5, # Probability of applying transformation
+    draw:(int, [int], callable)=None, # User defined behavior of applying batch transformation
+    size:(int, tuple)=None, # Output size, duplicated if one value is specified
+    mode:str='bilinear', # `F.grid_sample` mode ( 'bilinear', 'nearest', 'bicubic')
+    pad_mode=None, # A `PadMode`
+    batch=False, # Same behavior for the whole batch if `True`
+    align_corners=True # `F.grid_sample` align_corners
+):
     x0,mode,pad_mode = _get_default(x, mode, pad_mode)
     mat = _prepare_mat(x, dihedral_mat(x0, p=p, draw=draw, batch=batch))
     return x.affine_coord(mat=mat, sz=size, mode=mode, pad_mode=pad_mode, align_corners=align_corners)
 
 # Cell
 class Dihedral(AffineCoordTfm):
     "Apply a random dihedral transformation to a batch of images with a probability `p`"
-    def __init__(self, p=0.5, draw=None, size=None, mode='bilinear', pad_mode=PadMode.Reflection, align_corners=None, batch=False):
+    def __init__(self,
+        p=0.5, # Probability of applying transformation
+        draw:(int, [int], callable)=None, # User defined behavior of applying batch transformation
+        size:(int, tuple)=None, # Output size, duplicated if one value is specified
+        mode:str='bilinear', # `F.grid_sample` mode ( 'bilinear', 'nearest', 'bicubic')
+        pad_mode=PadMode.Reflection, # A `PadMode`
+        batch=False, # Same behavior for the whole batch if `True`
+        align_corners=True # `F.grid_sample` align_corners
+    ):
         f = partial(dihedral_mat, p=p, draw=draw, batch=batch)
         super().__init__(aff_fs=f, size=size, mode=mode, pad_mode=pad_mode, align_corners=align_corners)
 
 # Cell
 class DeterministicDihedral(Dihedral):
-    def __init__(self, size=None, mode='bilinear', pad_mode=PadMode.Reflection, align_corners=None):
+    def __init__(self,
+        size:(int, tuple)=None, # Output size, duplicated if one value is specified
+        mode:str='bilinear', # `F.grid_sample` mode ( 'bilinear', 'nearest', 'bicubic')
+        pad_mode=PadMode.Reflection, # A `PadMode`
+        align_corners=None # `F.grid_sample` align_corners
+    ):
         "Flip the batch every other call"
-        super().__init__(p=1., draw=DeterministicDraw(list(range(8))), pad_mode=pad_mode, align_corners=align_corners)
+        super().__init__(p=1., size=size, draw=DeterministicDraw(list(range(8))), pad_mode=pad_mode, align_corners=align_corners)
```

Comment:
No code changes in docs PRs. If you think this is a bug, please open an issue about it in the [fastai repo](https://github.com/fastai/fastai).
```suggestion
        super().__init__(p=1., draw=DeterministicDraw(list(range(8))), pad_mode=pad_mode, align_corners=align_corners)
```